### PR TITLE
Blog App : Controllers

### PR DESCRIPTION
**Pull request by Piercel2022**
For each URLs on the wireframe below   I create:

    A route.
    An action in the correct controller.
    A view file.

blog_root_users
blog_user_page
blog_user_all_posts
blog_single_post

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -3,4 +3,8 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  root 'users#index'
+  get 'users/:id', to: 'users#show'
+  get 'users/:user_id/posts' , to: 'posts#index'
+  get 'users/:user_id/posts/:id' , to: 'posts#show'
 end
```
Comment (algerina) [inline config/routes.rb]: - [ Optional] I recommend keeping your routes simple for now and using something like: 

`
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end

`

File: README.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 <!-- PROJECT DESCRIPTION -->
 
 # 📖  <a name="about-project">blog app</a>
-The project test validations and models of the Blog application with the rspec.
+The project implements the two differents controllers and the correponding actions in the view.
 
 **blog app** is a blog application
```
Comment (algerina) [inline README.md]: - [ ] Kindly use Microverse Readme template without deleting the first section. It is a requirement in all the projects.

**Review by algerina — CHANGES_REQUESTED** [review]
Hi @Piercel2022 ,

Great job so far! 
However, you still have minor issues to fix.

May I highlight the following: 
- [x] Your project looks awesome👏  ✔️ 
- [x] Your PR is professional and well detailed✔️ 
- [x] Well done implementing most of the requirements✔️ 



## STATUS: CHANGES REQUIRED ♻️ 
PLEASE: check the comments under the review



 
 ## Optional suggestions

 - Please check the comment under the review


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread and tag me @algerina if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)
___
_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [Algerina](https://github.com/algerina), I will receive a notification and will respond._

[GitHub](https://github.com/algerina) | [LinkedIn](https://linkedin.com/in/amel-khiri) | [Twitter](https://twitter.com/AmelQahwadji)

_Reviewed by [Amel Khiri ](https://github.com/algerina)_

**Comment by arslanbisharat** [comment]
Hi @Piercel2022,
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- The Pull Request has a proper title and description. ✔️ 
- You have added a descriptive Readme file. Good Job. ✔️ 
- There is no linter error is present. ✔️ 
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**